numpy/_core/tests/test_umath.py

```diff
@@ -2443,6 +2444,23 @@ def test_precision(self):
                 assert_equal(np.maximum([v1], [v2]), [expected])
                 assert_equal(np.maximum.reduce([v1, v2]), expected)
 
+    def test_maximum_too_many_args(self):
+        a = np.array([1])
+        b = np.array([2])
+        out = np.empty_like(a)
+        with pytest.warns(DeprecationWarning, match=r"Passing more than 2 positional arguments to np\.maximum is deprecated"):
+            result = np.maximum(a, b, out)
+        np.testing.assert_array_equal(result, [2])
+
+    def test_minimum_too_many_args(self):
+        a = np.array([1])
+        b = np.array([2])
+        out = np.empty_like(a)
+
+        with pytest.warns(DeprecationWarning, match=r"Passing more than 2 positional arguments to np\.minimum is deprecated"):
+            result = np.minimum(a, b, out)
```

Comment:
Sorry for not flagging earlier, but please move this test (following the pattern there) to `test_deprecations.py`.

In general I don't care too much about it, but I do care about testing the error path when the warning is raised as an error, and that is not tested here, but the pattern in the file always tests it.

(And when you do that, you should see that the C code is missing error checks and the test fails in weird ways.)